### ISSUE TITLE
Linux Mint Cinnamon (Nemo) integration

### ISSUE THREAD

**Issue by idielcfg** [alphaonex86/Ultracopier#57]
Hello i love ultracopier, i think it is a tool that most linux desktop users appreciate. In my case i use Linux Mint 20.04 and it would be grate if it can be integrated with Nemo. There is any possibility to have these in future version?

**Comment by alphaonex86** [comment]
Hi,
I have already requested to Nemo, but for now refused by Nemo.
You can request it again if you want

**Comment by idielcfg** [comment]
thanks i will request it too, thanks

**Comment by rambo919** [comment]
Seems that Nemo complains about qt being the major reason it "won't work"..... what about dolphin then instead for qt DE's like KDE?

**Comment by alphaonex86** [comment]
no, all the base code is C, the unix socket is just a unix socket, and work with other file manager in Qt, GTK, windows, ...
I my case I daily use with PCMan File Manager and PCMan File Manager Qt.

**Comment by ernzo** [comment]
I just published [this modification](https://github.com/ernzo/Nemo-UltraCopier/blob/patch-2/nemo-ultracopier.py) of [Nemo-Ultracopier](https://github.com/acidburn0zzz/Nemo-UltraCopier), which fixes the integration between Nemo and Ultracopier, and uses CopyQ to make the actual copy.

Caveat is: initally you'll need to perform the Copy/Paste operation twice because, 
the first time you run the script CopyQ will not be running (unless you launch it manually), and so it won't be able to capture the clipboard contents..

The second time you perform Copy/"Paste with Ultracopier" copyq will be running, and the script shall work as intended.

It works allright here with: Debian (trixie) + KDE Plasma/Wayland + latest stock Nemo + Ultracopier + CopyQ. 
(Gnome and Mate are installed too but not running)

https://github.com/ernzo/Nemo-UltraCopier/blob/patch-2/nemo-ultracopier.py


-Btw this was made by torturing ChatGPT for hours until it made it right, or almost..

The first version did perform the copy/paste allright, but failed to launch CopyQ on its own, so it needed to be launched manually.

Then I asked Copilot to fix the script, and it made one that could launch copyq automatically on its own.. 
but failed to perform the actual copy/paste.

So I went back to ChatGPT and asked to integrate one thing into the other..
And well, this is the final result!

All credits to the Original Developer/s, Lester, and ChatGPT + Copilot who did the rough work..

This is probably not a definitive solution, 
but maybe there's something we can learn about it, and can be implemented in a more definitive solution.

**Comment by alphaonex86** [comment]
ok, how use it?
excuting it say me:

**Comment by ernzo** [comment]
Well, first you need to install [Nemo-Ultracopier](https://github.com/acidburn0zzz/Nemo-UltraCopier), (and CopyQ)
and then replace the original nemo-ultracopier.py with the new version.. located at /home/USER/.local/share/nemo/actions/

Then use chmod to grant permission to execute the file /home/USER/.local/share/nemo/actions/nemo-ultracopier.py

The regular installation procedure is detailed in the Nemo-Ultracopier itself:
https://github.com/acidburn0zzz/Nemo-UltraCopier

**Comment by alphaonex86** [comment]
Ok, I miss reading something, my file manager is dolphin, then not apply.

**Comment by ernzo** [comment]
I just published a similar Action script for Dolphin..
it uses CopyQ to Paste the clipboard contents with Ultracopier.

https://github.com/ernzo/Dolphin-UltraCopier

It works perfecly in Debian (trixie) Plasma/Wayland + latest stock CopyQ & Ultracopier.
